static/app/views/issueList/groupSearchViewTabs/draggableTabBar.tsx

```diff
@@ -346,7 +347,7 @@ export function DraggableTabBar({
               ...queryParams,
               query: tab.unsavedChanges?.[0] ?? tab.query,
               sort: tab.unsavedChanges?.[1] ?? tab.querySort,
-              ...(tab.id !== 'temporary-tab' ? {viewId: tab.id} : {}),
+              viewId: tab.id !== 'temporary-tab' ? tab.id : undefined,
```

Comment:
explicitly remove viewId from search params if temp tab is clicked. Previously, this would just persist whatever view's id you were on last.